packages/template-typescript-minimal/package.json

```diff
@@ -11,7 +11,7 @@
     "lint:js": "pwa-kit-dev lint \"**/*.{js,jsx,ts,tsx}\"",
     "push": "npm run build && pwa-kit-dev push",
     "save-credentials": "pwa-kit-dev save-credentials",
-    "start": "pwa-kit-dev start",
+    "start": "cross-env NODE_ICU_DATA=node_modules/full-icu pwa-kit-dev start",
```

Comment:
curious why is this change needed?

Reply:
I was going through some of the dependencies and seeing what could be removed. Full-ICU was one of the packages that I was going to remove because it wasn't being used. I checked to see what it does and why, and there is still the chance that we might actually want it. So instead of removing it, I actually use it like we are in the retail-react-app template.